tests/functional/test_contract_instance.py

```diff
@@ -26,6 +27,11 @@ class DataObject(BaseModel):
     return DataObject()
 
 
+def test_use_contract(Contract, address):
```

Comment:
ape fixtures are not available in core's tests like this, do remove this and only use the integration test